core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -51,6 +69,10 @@ public MissingQueryBuilder nullValue(boolean nullValue) {
         return this;
     }
 
+    public boolean nullValue() {
```

Comment:
good point, actually given that it's a public method some javadocs wouldn't hurt